templates/source/access-points.html

```diff
@@ -117,8 +117,17 @@
                 </div>
             </div>
             <div class="form-group">
-                <div class="col-sm-12">
+                <div class="col-sm-12 text-right">
                     <br />
+                    <button type="button" class="btn btn-danger cancel-button">
+                      {% if accesspoint.uuid %}
+                        <a href="{% url 'view-access-point' accesspoint.uuid %}">
+                      {% else %}
+                        <a href="{% url 'add-access-point' source.uuid %}">
+                      {% endif %}
```

Comment:
What's this conditional flow attempting to accomplish? I can't think of a situation where we wouldn't want this button to link to `{% url 'view-access-point' source.uuid %}`. (The above suggestion changing the button styles incorporates this feedback, but if conditional flow is indeed necessary here then you should feel free to implement the suggestion above on your own.)

Reply:
i had the logic in to account for the case where `accesspoint` didn't exist. that's fixed and this logic is eliminated by using `source.uuid` instead!